spec/models/setting_spec.rb

```diff
@@ -53,6 +53,11 @@
       expect(homepage_setting.type).to eq "homepage"
     end
 
+    it "returns the key prefix for 'sdg' settings" do
+      sdg_setting = Setting.create!(key: "sdg.whatever")
```

Comment:
Add a blank line before the `expect` :pray:, so the test and the expectation are (kind of) separated.

Reply:
This was also consistent with the rest of the file 😅 , but obviously better to get it right 👌